doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,19 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info <vol_name>
+
+The output format is json and contains fields as follows.
+
+* volume_name: display the CephFS volume name
+* pools: data and metadata pools related information
+* mon_addrs: list of monitor addresses
+* size: current used size of the CephFS volume
+* subvolumegroups: number of subvolumegroups
```

Comment:
This is useful in scale setup's. we don't need to list all the subvolumegroups to check any subvolumegroup exists in a filesystem. will this also consider the internal subvolumegroups like `_index` `_deleting` `legacy` etc?

some discussion around same area here https://github.com/rook/rook/pull/9915#discussion_r881036730  https://github.com/rook/rook/pull/9915#discussion_r881225141
can we also add the same info CLI to the subvolumegroup also? 

```bash
 $ ceph fs subvolumegroup info <vol_name> <group_name>
```

Reply:
@kotreshhr is working on `$ ceph fs subvolumegroup info <vol_name> <group_name>`
@Madhu-1 I have added comment https://github.com/ceph/ceph/pull/44347#issuecomment-1137538847